EXT_SpeciesType.md

```diff
@@ -0,0 +1,80 @@
+Convention for Specifying Particle Species 
+==========================================
+
+openPMD extension name: `SpeciesType`
+
+openPMD extension ID: `4`
+
+
+Introduction
+------------
+
+This convention is for standardizing the names of particle species, e.g. in
+particle physics.
+
+
+Additional Record Attribute
+---------------------------
+
+The following additional attribute for openPMD `records` is defined in this
+extension:
+
+- `SpeciesType`
+  - type: *(string)*
+  - scope: *recommended*
```

Comment:
offline discussion: maybe better *optional* here, it's weird for e.g. "E fields" or arbitrary other records

Later on, in the `ED-PIC` extension or the `BeamPhysics` extension one can still specify, that it is *recommended* for e.g. `particle species`.